scormxblock/scormxblock.py

```diff
@@ -282,7 +282,7 @@ def get_context_student(self):
             scheme = 'https' if settings.HTTPS == 'on' else 'http'
             scorm_file_path = '{}://{}{}'.format(
                 scheme,
-                configuration_helpers.get_value('site_domain', settings.ENV_TOKENS.get('LMS_BASE')),
+                configuration_helpers.get_value('site_domain', settings.LMS_BASE),
```

Comment:
Could you describe a bit this fix? What was the issue?

I've quickly checked this on our Lilac-rg:

```python
>>> settings.ENV_TOKENS.get('LMS_BASE')
'lms-lilac-legacy.raccoongang.com'
>>> settings.LMS_BASE
'lms-lilac-legacy.raccoongang.com'
```

Reply:
I faced the issue when running it on devstack, it's not convenient to edit `lms.yml` or `lms.env.json` in order to change  
 this variable durring local development.

Reply:
settings.ENV_TOKENS.get('LMS_BASE', settings.LMS_BASE)